### DISCUSSION TITLE
a new and feature rich plugin / preset

### DISCUSSION THREAD

**Discussion by aliibrahim123** [GrapesJS/grapesjs#3844] [Ideas]
i made a new plugin / preset, [here it is](https://github.com/aliibrahim123/grapesjs-pp)
it has so many features like semi full html5 and css3 support and many panel like page manager and ace and method draw and 
general html component editing and search blocks and many others
please try it

**Comment by artf** [comment]
@aliibrahim123 add a demo link to your repo so people can actually try it

**Comment by aliibrahim123** [comment]
a [demo](https://aliibrahim123.github.io/grapesjs-pp/index.html)
